api/src/auth/redirect-exception.filter.ts

```diff
@@ -0,0 +1,14 @@
+import { ArgumentsHost, Catch, ExceptionFilter, HttpException } from '@nestjs/common';
+import { Response } from 'express';
+
+@Catch()
+export class HttpExceptionFilter implements ExceptionFilter {
+  catch(exception: HttpException, host: ArgumentsHost) {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response>();
+    const request = ctx.getRequest<Request>();
+    
+    response.redirect('http://localhost:3000/login?code=' + exception.name);
```

Comment:
url in .env

Reply:
Commit dae2f0e use the url from a env var